### ISSUE TITLE
[Doc] Write your own GNN module -- Example Code Improvement

### ISSUE THREAD

**Issue by Devansh-Agarwal** [dmlc/dgl#7856]
## 📚 Documentation

Tutorial part: [Write your own GNN module ](https://www.dgl.ai/dgl_docs/tutorials/blitz/3_message_passing.html#write-your-own-gnn-module)

In the [Training Loop code](https://www.dgl.ai/dgl_docs/tutorials/blitz/3_message_passing.html#training-loop), there is a variable `all_logits`, which is present in the code and is accumulating the `logits` at each epoch but this accumulation is not being used anywhere. 

I think it is better to remove `all_logits` from the code rather than keeping it as it causes confusion to the new users, it makes them think why are the logits being accumulated, when they are never being used.

If this seems an improvement, I can do a PR with the changes.
